test/support/test_helpers.ex

```diff
@@ -1,15 +1,33 @@
 defmodule RemoteRetro.TestHelpers do
   use Wallaby.DSL
-  alias RemoteRetro.Repo
+  alias RemoteRetro.{Repo, User}
 
   def persist_idea_for_retro(context) do
-    %{idea: idea, retro: retro} = context
-    idea = Map.put(idea, :retro_id, retro.id)
+    %{idea: idea, retro: retro, user: user} = context
+    user =
+      if Map.has_key?(user, :id) do
+        user
+      else
+        Repo.get_by(User, email: user["email"])
+      end
+    idea =
+      Map.put(idea, :retro_id, retro.id)
```

Comment:
Rather than call `Map.put/3` twice, you could use [`Map.merge/2`](https://hexdocs.pm/elixir/Map.html#merge/2)